### PR TITLE
Improves Croatian support.

### PR DESCRIPTION
**Pull request by cyanic-selkie**
This PR adds better support for Croatian. Mainly improves morphological variant coverage and adds a few additional relative forms.

### CODE REVIEW

File: dateparser/data/date_translation_data/hr.py [hunk 1]
```diff
@@ -4,62 +4,77 @@
     "january": [
         "sij",
         "siječanj",
-        "siječnja"
+        "siječnja",
+        "siječnju"
```
Comment (LRepalust) [inline dateparser/data/date_translation_data/hr.py:8]: I would add siječnjem here

File: dateparser/data/date_translation_data/hr.py [hunk 2]
```diff
@@ -4,62 +4,77 @@
...
     ],
     "february": [
         "velj",
         "veljača",
-        "veljače"
+        "veljače",
+        "veljači"
```
Comment (LRepalust) [inline dateparser/data/date_translation_data/hr.py:14]: I would add veljačom here

File: dateparser/data/date_translation_data/hr.py [hunk 3]
```diff
@@ -4,62 +4,77 @@
...
     ],
     "march": [
         "ožu",
         "ožujak",
-        "ožujka"
+        "ožujka",
+        "ožujku"
```
Comment (LRepalust) [inline dateparser/data/date_translation_data/hr.py:20]: I would add "ožujkom" here

File: dateparser/data/date_translation_data/hr.py [hunk 4]
```diff
@@ -4,62 +4,77 @@
...
     ],
     "april": [
         "tra",
         "travanj",
-        "travnja"
+        "travnja",
+        "travnju"
```
Comment (LRepalust) [inline dateparser/data/date_translation_data/hr.py:26]: I would add travnjem here

File: dateparser/data/date_translation_data/hr.py [hunk 5]
```diff
@@ -4,62 +4,77 @@
...
     ],
     "may": [
         "svi",
         "svibanj",
-        "svibnja"
+        "svibnja",
+        "svibnju"
```
Comment (LRepalust) [inline dateparser/data/date_translation_data/hr.py:32]: I would add "svibnjem" here

File: dateparser/data/date_translation_data/hr.py [hunk 6]
```diff
@@ -4,62 +4,77 @@
...
     ],
     "june": [
         "lip",
         "lipanj",
-        "lipnja"
+        "lipnja",
+        "lipnju"
```
Comment (LRepalust) [inline dateparser/data/date_translation_data/hr.py:38]: I would add "lipnjem" here

File: dateparser/data/date_translation_data/hr.py [hunk 7]
```diff
@@ -4,62 +4,77 @@
...
     ],
     "july": [
         "srp",
         "srpanj",
-        "srpnja"
+        "srpnja",
+        "srpnju"
```
Comment (LRepalust) [inline dateparser/data/date_translation_data/hr.py:44]: I would add "srpnjem" here

File: dateparser/data/date_translation_data/hr.py [hunk 8]
```diff
@@ -4,62 +4,77 @@
...
     ],
     "august": [
         "kol",
         "kolovoz",
-        "kolovoza"
+        "kolovoza",
+        "kolovozu"
```
Comment (LRepalust) [inline dateparser/data/date_translation_data/hr.py:50]: I would add "kolovozom" here

File: dateparser/data/date_translation_data/hr.py [hunk 9]
```diff
@@ -4,62 +4,77 @@
...
     ],
     "september": [
         "ruj",
         "rujan",
-        "rujna"
+        "rujna",
+        "rujnu"
```
Comment (LRepalust) [inline dateparser/data/date_translation_data/hr.py:56]: I would add "rujnom" here

File: dateparser/data/date_translation_data/hr.py [hunk 10]
```diff
@@ -4,62 +4,77 @@
...
     ],
     "october": [
         "lis",
         "listopad",
-        "listopada"
+        "listopada",
+        "listopadu"
```
Comment (LRepalust) [inline dateparser/data/date_translation_data/hr.py:62]: I would add "listopadem" here

File: dateparser/data/date_translation_data/hr.py [hunk 11]
```diff
@@ -4,62 +4,77 @@
...
     ],
     "november": [
         "stu",
         "studeni",
-        "studenoga"
+        "studenoga",
+        "studenom",
+        "studenomu",
+        "studenome",
+        "studenog"
```
Comment (LRepalust) [inline dateparser/data/date_translation_data/hr.py:71]: I would add "studenim" here

File: dateparser_data/supplementary_language_data/date_translation_data/hr.yaml [hunk 12]
```diff
@@ -0,0 +1,133 @@
...
+    - kolovozu
+september:
+    - rujnu
+october:
+    - listopadu
+november:
+    - studenom
+    - studenomu
+    - studenome
+    - studenog
```
[the remark this answers is not in the document]
Comment (cyanic-selkie) [reply dateparser_data/supplementary_language_data/date_translation_data/hr.yaml:38]: The default forms already include "studenoga", this is an alternative acceptable spelling. While "u studenog" doesn't mean anything, "12. studenog(a) 2022." does and is a very common date form in Croatian.

[the remark this answers is not in the document]
Comment (LRepalust) [reply dateparser_data/supplementary_language_data/date_translation_data/hr.yaml:38]: I agree with this. I was thinking that all of these are connected with "in" (u) in this part, my bad.

File: dateparser/data/date_translation_data/hr.py (hunk 10 above)
Comment (LRepalust) [reply dateparser/data/date_translation_data/hr.py]: You assume right.

**Review by LRepalust — APPROVED** [review]
LGTM

**Review by serhii73 — APPROVED** [review]

**Review by Gallaecio — APPROVED** [review]

**Comment by Gallaecio** [comment]
Thanks!

**Pull request merged**